### ISSUE TITLE
ComputerCraft/CC:Tweaked Computers dont save current state to memory

### ISSUE THREAD

**Issue by PaxiTheAlchemist** [EnigmaticaModpacks/Enigmatica6#3631]
### Modpack Version

0.5.17

### Describe your issue.

# Description

## TL:DR

Computers from the ComputerCraft/CC: Tweaked mod will automatically power off when a chunk unloads.

## What is the issue?

When running a program on a computer or simply booting one up, it would seem like the computers will shut down when the chunk they are located within becomes unloaded or when the server restarts. The expected behavior of the computer is that it saves its current state whenever the game shuts down or the chunk it is within is unloaded. When reloaded, either when the chunk is loaded once more or the server boots up, it should recall its previous state from memory and continue from where it left off. During the time when force chunk loading was cleared on top of whenever the server restarted (On the [official Enigmatica 6 server](https://discord.gg/enigmatica) server I might add), I would notice that all my computers would shut off and would have to be restarted. When noticing this issue, I contacted an admin of the server to which they declared that there is no configuration within the server that would alter the behavior of this feature.

### Have you modified the modpack?

Yes

### User Modifications

# I do not find this essential to my issue

Due to the fact that this issue occurs on the [official Enigmatica 6 server](https://discord.gg/enigmatica) which has little to no modification to the pack to which users joining would have to share the essential mods of the server to join it, on top of reporting to an admin of said server who declared that there are no configurations that would modify the described expected behavior above, I feel like the modification to my pack would have little to no influence on the issue

## Modifications

In any case, the modifications to my mod pack include only quality of life mods including [Optifine](https://optifine.net), [BetterF3](https://www.curseforge.com/minecraft/mc-mods/betterf3), [Classic Bars](https://www.curseforge.com/minecraft/mc-mods/classic-bars), and more. Nothing that would interact with the server itself or the ComputerCraft/CC: Tweaked mod, addons, and or its components

### Did the issue happen in singleplayer or on a server?

Server

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue. 
I agree that your modifications should not play a role here 👍
